### DISCUSSION TITLE
Is this project still being maintained/developed

### DISCUSSION THREAD

**Discussion by tomvanschaijk** [pcah/python-clean-architecture#84] [General]
I came across this project as I was searching for people that already tried designing and implementing a clean architecture in Python. I mainly worked in C# for the last decades and recently started at a company using mainly Python. Of course I wanted to stick to good architectural principles, so I started struggling with Python to get it all done, and then came across this project. I did notice, however, that there haven't been any updates for about 2 years. Did the project move/got shelved/..?

In any case: I love the effort and the goal of the project. I do love many aspects of Python, and I believe that (like you said) with some effort it really CAN be used to develop clean architectures. It doesn't only have to be a nice & easy scripting glue language.

**Comment by lhaze** [comment]
Hi all

I am the maintainer of this repo. Thank you for the above question and, more generally, for your interest in the topic. I was going to answer right after it was asked, but the comment got rather long, and at the same time, it happened right before my extended trip. Unfortunately, it took me a little too long to get back to the answer.

I will give a longer info in the coming days.

**Reply by tomvanschaijk** [reply]
Thank you! That's appreciated ;-)
